src/input/SimModSuite.h

```diff
@@ -357,6 +357,54 @@ struct get_second : public std::unary_function<MyMap::value_type, int>
     }
 };
 
+enum elementType {
+  vertex = 0,
+  edge = 1,
+  face = 2,
+  region = 3
+};
+
+void setMeshSize(pGModel model, pACase &meshCase, MeshAttributes &MeshAtt) {
+    // Set mesh size on vertices, edges, surfaces and regions
+    const char *element_name[4] = { "vertex", "edge", "face", "region" };
+
+    for (int elType = vertex; elType != region+1; elType++ )
+    {
+        std::list <std::pair <std::list<int>, double>> lpair_lId_MSize;
```

Comment:
Could be a reference. But then you'd need something like
`const std::list<...>& lpair_lId_MSize  = MeshAtt.getMSize(elType);`
with `const std::list<...>& MeshAtt::getMSizeList(elementType type)` and then have the switch in there.

Reply:
Thx for the suggestion. But then I get:
```
/hppfs/work/pr45fi/di73yeq4/myLibs/PUMGen/src/input/MeshAttributes.h(68): error: identifier "ElementType" is undefined
      const std::list <std::pair <std::list<int>, double>>& getMSizeList(ElementType eltype);
```
 

the enum class is defined in "src/input/SimModSuite.h". 
I tried adding #include "SimModSuite.h" at the beginning of MeshAttributes.h.
But there must be somewhere else to be changed (cMake related file?).
Any hint?